.github/workflows/continuous-integration-workflow.yml

```diff
@@ -104,4 +85,4 @@ jobs:
     - name: Running tests
       run: ./tests/node_modules/.bin/mocha tests/ --exit --require tests/hooks.js --config tests/.mocharc.js "$@"
       env:
-        LEDGER_APP: bin/app.elf
+        LEDGER_APP: bin/nervos_nanos.elf
```

Comment:
Functional tests only for nanos :cry: ?

Reply:
Yes, but the external dev is apparently working on new Ragger tests.